src/components/Table/_pagination.scss

```diff
@@ -55,14 +55,14 @@ html[dir='rtl'] {
 
   .#{$prefix}--pagination__right .#{$prefix}--pagination__text {
     margin-left: $spacing-05;
-    margin-right: 0.0625rem;
+    margin-right: $spacing-01 - 0.0625rem;
```

Comment:
Not quite sure what we are doing here. Looks like you are subtracting the original value from the token. Is this intentional?